tests/system/providers/google/ads/example_ads.py

```diff
@@ -61,28 +68,58 @@
     "metrics.all_conversions.value",
     "metrics.cost_micros.value",
 ]
-
 # [END howto_google_ads_env_variables]
 
 with models.DAG(
-    "example_google_ads",
-    schedule_interval=None,  # Override to match your needs
+    DAG_ID,
+    schedule_interval='@once',
```

Comment:
Is this change necessary?

Reply:
This is mentioned in the AIP-47 **Design details**.